### PR TITLE
Init French translation

### PR DESCRIPTION
**Pull request by sieben**

### CODE REVIEW

**Comment by sieben** [comment]
I've signed the individual agreement

**Comment by sieben** [comment]
@perriea @rekcah78 would like to contribute as well to the French translation.

File: README-fr.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+make docker-serve
+```
+
+Ouvrez votre navigateur à l'adresse: http://localhost:1313 pour voir le site.
+Lorsque vous apportez des modifications aux fichiers sources, Hugo met à jour le site et force le navigateur à rafraîchir la page.
+
+## Exécuter le site localement en utilisant Hugo
+
+Voir la [documentation officielle Hugo](https://gohugo.io/getting-started/installing/) pour les instructions d'installation Hugo.
+Assurez-vous d'installer la version Hugo spécifiée par la variable d'environnement `HUGO_VERSION' dans le fichier[`netlify.toml`](netlify.toml#L9).
```
Comment (lucperkins) [inline README-fr.md]: Change `\`HUGO_VERSION'` to `\`HUGO_VERSION\``

Comment (sieben) [reply README-fr.md]: done

File: OWNERS_ALIASES [hunk 2]
```diff
@@ -137,6 +137,24 @@ aliases:
     - stewart-yu
     - xiangpengzhao
     - zhangxiaoyu-zidif
+  sig-docs-fr-owners: #Team: Documentation; GH: sig-docs-fr-owners
```
Comment (zacharysarah) [inline OWNERS_ALIASES]: Excellent--thank you for adding these aliases. ✨

**Comment by zacharysarah** [comment]
@sieben You will also need to add a file: `/content/fr/OWNERS`. For comparison, take a look at the English version: https://github.com/kubernetes/website/blob/master/content/en/OWNERS

**Comment by zacharysarah** [comment]
Test:
/language fr

**Comment by sieben** [comment]
@zacharysarah I think I've added the relevant block to `config.toml` Could you tell me what is missing?

**Comment by zacharysarah** [comment]
@lucperkins 👋 Do you mind reviewing this for a LGTM?

/approve

**Comment by zacharysarah** [comment]
@sieben 
> I think I've added the relevant block to `config.toml`
> Could you tell me what is missing?

It looks good to me. 💯

**Comment by sieben** [comment]
How could it be merged now?

**Comment by zacharysarah** [comment]
@sieben Just waiting on approval from a second reviewer. I've tagged @lucperkins; he'll get to it as his time permits.

**Comment by yastij** [comment]
/hold

until @lucperkins reviews, I'll try to do a pass also on this.

**Comment by zparnold** [comment]
YAAAAAS!! French!!!

**Comment by sieben** [comment]
@yastij I think @lucperkins did a review previously.

**Comment by lucperkins** [comment]
LGTM

Or perhaps more appropriately TTBT (très très bon travail)

**Pull request merged**